src/gh_pr/ui/interactive.py

```
@@ -0,0 +1,770 @@
+            self.prs.sort(key=lambda pr: pr.get("number", 0), reverse=True)
+        elif sort_option == "author":
+            self.prs.sort(key=lambda pr: pr.get("author", "").lower())
+        elif sort_option == "created":
+            self.prs.sort(key=lambda pr: pr.get("created_at", ""), reverse=True)
+        elif sort_option == "updated":
+            self.prs.sort(key=lambda pr: pr.get("updated_at", ""), reverse=True)
+
+        # Refresh the display
+        self.update_pr_list()
+
+    def handle_export(self, export_format: str, filename: str = None) -> None:
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)